components/ai_chat/core/common/test_mojom_printers.cc

```diff
@@ -51,10 +51,19 @@ void PrintTo(const mojom::ToolUseEvent& event, std::ostream* os) {
   *os << "--ToolUseEvent--\n";
   *os << "tool_name: " << event.tool_name << "\n";
   *os << "id: " << event.id << "\n";
-  *os << "arguments: ";
-  std::string args_json;
-  base::JSONWriter::Write(event.arguments, &args_json);
-  *os << args_json << "\n";
+  *os << "arguments_json: " << event.arguments_json << "\n";
+  if (event.arguments) {
+    *os << "arguments (stored as parsed ): ";
+    std::string args_json;
+    base::JSONWriter::Write(event.arguments.value(), &args_json);
+    *os << args_json << "\n";
+  }
```

Comment:
is this going to make the test output a bit too verbose? Maybe lets just write the `arguments_json` as if that matches then the parsed value should also match, right?

Reply:
Not that it's relevant after I remove the parsed arguments property - but I'd say it would have been correct to keep both since some of the tests verify that they are the same. I imagine a subtle bug creeping in where the 2 properties become out of sync and a frustrated dev pointing at the screen yelling "but they are identical!".